.gitignore

```diff
@@ -3,3 +3,4 @@ install.rdf
 *.xpi
 node_modules
 addon/StudyUtils.jsm
+addon/PioneerUtils.jsm
```

Comment:
Same as [this comment](https://github.com/mozilla/shield-studies-addon-template/pull/49/files#r170994130); consider removing.